### PR TITLE
CNV-16295: Prevent NVIDIA GPU operands from deploying on nodes

### PR DESCRIPTION
**Pull request by ousleyp**
- [CNV-16295](https://issues.redhat.com/browse/CNV-16295)
- 4.14+
- preview build: https://42126--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough#virt-preparing-nodes-for-gpu-passthrough

### CODE REVIEW

**Comment by ousleyp** [comment]
> LGTM.
> 
> Btw, I am not sure if this should be in 4.10 already, as the vGPU specific Operator is neither tested nor available, AFAIK. I added LGTM assuming we do not want to install the existing NVIDIA GPU Operator which might already be available currently on marketplace, when using GPU Passthrough.

I was just about to close this PR because @vladikr relayed the message from @fabiand that this should not go into 4.10. However, I did change the wording here so that it does _not_ refer to vGPUs.

@fabiand, do you think that the text as it currently stands would benefit customers in 4.10 (based on Kedar's comment)? 

Thanks!

**Comment by ousleyp** [comment]
Closing. Might reopen in 4.11.

**Comment by kbidarkar** [comment]
LGTM

File: virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc [hunk 1]
```diff
@@ -17,10 +17,22 @@ The Peripheral Component Interconnect (PCI) passthrough feature enables you to a
 
 Cluster administrators can expose and manage host devices that are permitted to be used in the cluster by using the `oc` command-line interface (CLI).
 
-include::modules/virt-about-pci-passthrough.adoc[leveloffset=+1]
+[id="virt-preparing-nodes-for-gpu-passthrough"]
+== Preparing nodes for GPU passthrough
+
```
Comment (bscott-rh) [inline virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc:22]: You may want to add a short abstract on line 22 before going right into the subheading.

**Review by bscott-rh — APPROVED** [review]
I left 2 comments for your consideration - otherwise LGTM.

File: modules/virt-preventing-nvidia-gpu-operands-from-deploying-on-nodes.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-preventing-nvidia-operands-from-deploying-on-nodes_{context}"]
+= Preventing NVIDIA GPU operands from deploying on nodes
+
+If you use the link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/contents.html[NVIDIA GPU Operator] in your cluster, you can prevent GPU operands from deploying on worker nodes that you use for GPU passthrough. Nodes that have the `nvidia.com/gpu.deploy.operands=false` label are not configured for GPU or vGPU because the pods that perform this configuration are not created. If these pods already existed, they are terminated when the node is labeled.
```
[the remark this answers is not in the document]
Comment (ousleyp) [reply modules/virt-preventing-nvidia-gpu-operands-from-deploying-on-nodes.adoc:10]: Love the suggestion, thank you!

**Pull request merged**